pyproject.toml

```diff
@@ -21,12 +21,27 @@ classifiers = [
 packages = [
     {include = "poetry"},
 ]
+include = [
+    "poetry/core/_vendor/**/*"
```

Comment:
As far as I know this is not necessary since it will included by default, won't it?

Reply:
Hmm, you might be right since we are greedy when we do the package lookup I think. Although, I have dropped the `__init__.py` file, figured I'd be explicit here.